src/applications/gi/components/profile/SchoolLocations.jsx

```diff
@@ -69,6 +69,8 @@ export class SchoolLocations extends React.Component {
       viewableRowCount: this.state.initialRowCount,
       viewAll: false,
     });
+    const schoolLocationsElement = document.getElementById('accordion-item-7');
+    schoolLocationsElement.scrollIntoView();
```

Comment:
The PR description doesn't mention this change and it also looks like it is for a different app.  Can you explain this?

Reply:
That was a mistake on my part from a previous change I was working on. The change was for a scrolling bug that was occurring in the child institution list on a Vet Tec profile page in the comparision tool.  I've pulled it out of this PR. Good Catch!